src/BMSController.h

```diff
@@ -0,0 +1,61 @@
+#include <Arduino.h>
+#include <Logger.h>
+
+#include "bms_relay.h"
+#include "packet.h"
+#include "settings.h"
+#include "ArduinoJson.h"
+#include "CanBus.h"
+
+//https://github.com/lolwheel/Owie/commit/8f10e0897eda7f6e3ca40afeaff909a97b9f7751
+
+// UART RX is connected to the *BMS* White line for OneWheel Pint.
+class BMSController {
+public:
+    BMSController();
+    void init(CanBus*);
+    void loop();
+    BmsRelay *relay;
+    HardwareSerial bms = HardwareSerial(1);  // Using UART1
```

Comment:
Couldn't this clash with other hardware / setups. I guess we need to make the usage of hardware-serial ports configurable.

Reply:
The actual pin is defined here as BMS_TX_PIN: https://github.com/thankthemaker/rESCue/pull/69/files#diff-d7f373856c5d54c6ae4baf547955e910104f6ff904f6e4bc697bf1787be5d585R22

We're using HardwareSerial(2); in main https://github.com/thankthemaker/rESCue/blob/master/src/main.cpp#L33

So I assume HardwareSerial(1) would also exist on ESP32 (and it does not look like it's being used)?

Maybe we need to make both usage of hardware-serial ports configurable?